src/WebJobs.Extensions.DurableTask/OrchestrationClientAttribute.cs

```diff
@@ -13,7 +12,11 @@ namespace Microsoft.Azure.WebJobs
     /// </summary>
     [AttributeUsage(AttributeTargets.Parameter)]
     [DebuggerDisplay("TaskHub={TaskHub}, ConnectionName={ConnectionName}")]
+#if NETSTANDARD2_0
+    [Binding(TriggerHandlesReturnValue = false)]
```

Comment:
`Binding.TriggerHandlesReturnValue` is only intended for trigger bindings and `OrchestrationClientAttribute` is a non-trigger binding, so I wouldn't expect this change to be necessary.

Reply:
It's not; the runtime won't look for the attribute on a non-trigger binding. This can be safely removed.